cpmonitor/models.py

```diff
@@ -651,7 +654,7 @@ class Meta:
         verbose_name_plural = "Diagramme"
 
     city = models.ForeignKey(City, on_delete=models.PROTECT, related_name="charts")
-    image = models.ImageField("Bilddatei", upload_to="uploads/%Y/%m/%d/")
+    image = models.ImageField("Bilddatei", upload_to="uploads/charts")
```

Comment:
The idea of this is to prevent directories from growing too large and to avoid file name collisions. I would rather prefer to have it for the featured image, as well. (But that is another issue.)

It might be more sensible to find another structuring, e.g. by city. However, this was a simple one-liner, since `ImageField` offers just this: https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.FileField.upload_to. Since it was so simple, I went ahead and used it, even if the above reasons might not seem imminent for our project.

I would prefer to keep the date in there. I would expect the migration to be simpler and certainly less error-prone. After all: It will run on the server during update and our procedure there is far from perfect.

Reply:
File collisions are not an issue. Django handles that transparently by adding unique identifiers.

A single directory is easier to handle when you want to run an operation on all files (such as resizing in this case). I think we do not need a structure by city. The folder does not have to be managed manually and if you want to get all files belonging to one city you just query the database. We could even put all files into a single folder, not even separate local groups and charts. I am still for a flat structure.

With regards to the migration: I expect there to be few images for the moment, so not too risky. If need be, I could even do it manually if I am given the necessary logins. Also, we can just make a copy of the upload folder before running the migration and restore it within seconds if anything should go wrong. Locally I tested it carefully.

Reply:
My experience is that directories with many entries tend to make problems on the OS side. Is that wisdom of the past?

Sure, single directory is easier to handle, but by how much? Walking a directory tree is no big deal, is it?

Reply:
> My experience is that directories with many entries tend to make problems on the OS side. Is that wisdom of the past?

I would be surprised if that was still an issue on modern Linux, but to be honest I do not know. Also, I thought that it is unlikely to have large amounts of images any time soon. Take 100 teams, 20 files each (which seems a lot), makes 2000, no big deal.

> Sure, single directory is easier to handle, but by how much? Walking a directory tree is no big deal, is it?

Fair enough. My impression was probably misleading because migrating the folder structure was a bit of a hassle. But a simple iteration is no big deal, you're right.

OK, I don't want to make this a big thing. You seem to strongly prefer the old structure, so I will revert to it. 
I would then, however, merge the local_group folder into the date folder structure, so we keep just a single pool of images and don't use different systems. Agreed?

Reply:
That would be very nice! You might as well do that in a separate PR, though, if you prefer. The icky part would be moving the images...